app/models/goal.py

```diff
@@ -1,6 +1,17 @@
 from flask import current_app
 from app import db
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship
```

Comment:
These imports shouldn't be needed as `ForeignKey` and `relationship` should be coming from the instance of sqlalchemy you instantiated in `__init__.py` called `db`.